jdisc_http_service/src/test/java/com/yahoo/container/logging/JsonConnectionLogWriterTest.java

```diff
@@ -1,7 +1,7 @@
 package com.yahoo.container.logging;// Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 import com.yahoo.test.json.JsonTestHelper;
-import org.junit.jupiter.api.Test;
+import org.junit.Test;
```

Comment:
You are effectively disabling all tests here now

Reply:
FYI `org.junit.jupiter.api` is junit5 while `org.junit` is junit4.

Reply:
This was not intended. That was just a temporary thing to get intellij to run tests in config-model. It did not recognize the jupiter.api.